projects/igniteui-angular/src/lib/services/overlay/position/IPositionStrategy.ts

```diff
@@ -17,5 +20,5 @@ export interface IPositionStrategy {
      * settings.positionStrategy.position(content, size, document, true);
      * ```
      */
-     position(contentElement: HTMLElement, size?: {}, document?: Document, initialCall?: boolean): void;
+    position(contentElement: HTMLElement, size?: { width: number, height: number }, document?: Document, initialCall?: boolean): void;
```

Comment:
This won't work because the **size** parameter is a object and when the comparison between the parameters came the objects would have different references which respectively means that the parameters are not equal.

Reply:
Fixed. Now we have Size interface and position method accepts Size parameter.